spot-client/src/common/routing/index.js

```diff
@@ -1 +1,2 @@
 export * from './constants';
+export { default as RestrictedRoute } from './restricted-route';
```

Comment:
+1, I'm not a fan of export renames either, makes it harder to track usages

Reply:
The name didn't change. Do you meant he casing?

Reply:
Yeah, you're probably talking about the file casing vs export casing. All files have been kebab cased since the beginning and no definitive decision was reached on how files should be cased. But for jitsi-meet consistency they'll be Title/camel cased and Pawel will start the renaming pass.

Reply:
Or maybe you're talking about how in restricted-route.js I have it named BrowserRestrictedRoute. The cause of that would be I went back and forth a lot on encapsulating the unsupported browser check within the custom route component or passing in the check. So the name change there would be unintentional.

Reply:
yes, about `BrowserRestrictedRoute` instead of `RestrictedRoute`